### PR TITLE
🚀 jats-to-myst

### PR DESCRIPTION
**Pull request by rowanc1**

### CODE REVIEW

File: packages/jats-to-myst/src/transforms/typography.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * ```
+ * [ ..., {paragraph: [child1, math1, child2, child3, math2]}, ... ]
+ * [ ..., {paragraph: [child1]}, math1, {paragraph: [child2, child3]}, math2, ... ]
+ * ```
+ *
+ * @param tree
+ * @param parentNodeType The parent (e.g. a paragraph)
+ * @param childNodeType The child (e.g. a list or math)
+ */
+function unnestTransform(tree: GenericParent, parentNodeType: string, childNodeType: string) {
```
Comment (rowanc1) [inline packages/jats-to-myst/src/transforms/typography.ts:21]: This is something that should eventually be reused in other places, came from myst-transforms.

File: packages/jats-to-myst/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+    case RefType.table:
+      return 'table';
+    case RefType.custom:
+      return undefined;
+  }
+}
+
+type Attributes = Record<string, any>;
+
+const handlers: Record<string, Handler> = {
```
Comment (rowanc1) [inline packages/jats-to-myst/src/index.ts:32]: Lots of things still to turn on here, but the following is going quite well on much of the JATS I have found.

**Review by rowanc1 — COMMENTED** [review]
Some notes.

**Review by fwkoch — APPROVED** [review]
🚀 Looks great!

File: packages/jats-to-myst/src/transforms/typography.ts (hunk 1 above)
Comment (rowanc1) [reply packages/jats-to-myst/src/transforms/typography.ts:21]: I think we should move this over to myst-transforms and deduplicate, then it should be good to go!

**Pull request merged**